src/deluge/deluge.cpp

```diff
@@ -452,23 +452,30 @@ void setupBlankSong() {
 /// Can only happen after settings, which includes default settings, have been read
 void setupStartupSong() {
 	auto startupSongMode = FlashStorage::defaultStartupSongMode;
```

Comment:
For safety against defaulting to a song that crashes the deluge, I think you could write a canary file here and delete it when the load is done